### PR TITLE
Update Discord, Stack Overflow, Leetcode Icons

### PR DESCRIPTION
**Pull request by Neesh774**
I noticed a few weird glitches with a few of the logos when transparency is applied to the bgColor, so I went into the SVGs and fixed them.

### CODE REVIEW

**Comment by hyo-livehive** [comment]
Thanks!

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-social-icons",
+  "name": "@neesh775/react-social-icons",
```
Comment (ibnumalik) [inline package.json:2]: ?

Comment (Jonak-Adipta-Kalita) [reply package.json:2]: The main changes should get implemented.... just that package.json thing is a bit....... messed up......

Comment (Neesh774) [reply package.json:2]: sorry, new to the whole forking/pr'ing thing

Comment (Neesh774) [reply package.json]: good now @ibnumalik

Comment (Neesh774) [reply package.json]: good now @ibnumalik

**Review by Jonak-Adipta-Kalita — APPROVED** [review]
👍

**Comment by couetilc** [comment]
hi @Neesh774, this has been addressed in [v6 of react-social-icons](https://www.npmjs.com/package/react-social-icons)

CC @ibnumalik @Jonak-Adipta-Kalita

**Comment by Jonak-Adipta-Kalita** [comment]
Got it

**Pull request closed without merging**